fe/fe-common/src/main/java/org/apache/doris/common/CatalogConfigFileUtils.java

```diff
@@ -52,7 +52,7 @@ private static <T> T loadConfigFromDir(String resourcesPath, String configDir,
         // Iterate over the comma-separated list of resource files.
         for (String resource : resourcesPath.split(",")) {
             // Construct the full path to the resource file.
-            String resourcePath = configDir + File.separator + resource.trim();
+            String resourcePath = configDir + resource.trim();
```

Comment:
It is best to determine whether the user's configuration contains file separators